confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -268,25 +378,22 @@ def delete_subject(self, subject):
             https://docs.confluent.io/current/schema-registry/develop/api.html#delete--subjects-(string-%20subject)
 
         """  # noqa: E501
-        self._rest_client.DELETE(url_formatter(SUBJECT, subject))
+        return self._rest_client.DELETE('subjects/' + subject)
```

Comment:
Try to be be consistent with how these urls are constructed, format or no format, but not both.